cli/cli_info.cpp

```diff
@@ -81,6 +81,13 @@ getApiName(int api) {
   return trace::API_NAMES[api];
 }
 
+static const std::string
+getProcessName(std::string name) {
+  if (name == "")
```

Comment:
`if (name.empty)()`  , but per comment below, we can omit this function entilrely.